includes/episode_number_column.php

```diff
@@ -25,3 +29,61 @@ function podlove_add_episodeno_column_content_to_episodes_table($column_name)
         }
 	}
 }
+
+function podlove_episodeno_quickedit_form($column_name)
+{
+    if ($column_name === 'episode_number') {
+			?>
+			<fieldset class="inline-edit-col-left">
+		    <div class="inline-edit-col">
+					<label class="alignleft">
+						<span class="title"><?php _e('Episode number', 'podlove-podcasting-plugin-for-wordpress') ?></span>
+						<input type="number" name="_podlove_meta[number]" class="podlove_meta_quickedit_episode_number" />
+					</label>
+					<?php wp_nonce_field( '_podlove_meta_quickedit_episode_number', '_podlove_meta_quickedit_episode_number_nonce_field' ); ?>
+		    </div>
+	    </fieldset>
+	    <?php
+		}
+}
+
+function podlove_episodeno_quickedit_save($post_id)
+{
+	if ( wp_verify_nonce($_POST['_podlove_meta_quickedit_episode_number_nonce_field'], '_podlove_meta_quickedit_episode_number') ) {
+		$episode = \Podlove\Model\Episode::find_one_by_post_id($post_id);
```

Comment:
Agree. At first glance I thought is_string() will do the job (basically it does: if the episode does not exist is_string() will return FALSE) but thinking beyond errors this will throw a notice if no episode object is returned.